manageiq-messaging.gemspec

```diff
@@ -29,4 +29,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "rubocop"
   spec.add_development_dependency "simplecov"
+  spec.add_development_dependency "oj"
```

Comment:
This is a C extension right?  If so, cc @simaishi

Reply:
@Fryguy this is just a development dependency. I can avoid if you prefer?